rn/Teacher/src/images/index.js

```diff
@@ -126,6 +126,7 @@ const images = {
   relax: require('./relax.png'),
   hamburger: require('./hamburger.png'),
   invisible: require('./invisible.png'),
+  off: require('./off.png')
```

Comment:
is this the eyeball?

Reply:
It is the eyeball with a line through it.

Reply:
![image](https://user-images.githubusercontent.com/4642727/64037486-b277cb80-cb23-11e9-937c-64780b9cdd26.png)

Reply:
ah, we don't have that one in inst-icons do we?  👍

Reply:
Unfortunately no. I thought of making a component that was the existing eyeball and a line but then decided to just add the icon and keep things simpler.